contracts/tokens/ERC20HoldableToken.sol

```diff
@@ -20,25 +20,15 @@ import "./IERC20HoldableToken.sol";
 contract ERC20HoldableToken is ERC20Token, IERC20HoldableToken {
     using SafeMath for uint256;
 
-    struct HoldData {
-        address sender;
-        address recipient;
-        address notary;
-        uint256 amount;
-        uint256 expirationDateTime;
-        bytes32 lockHash;
-        HoldStatusCode status;
-    }
-
     // mapping of accounts to hold data
-    mapping(bytes32 => HoldData) internal holds;
+    mapping(bytes32 => ERC20HoldData) internal holds;
     // mapping of accounts and their total amount on hold
     mapping(address => uint256) internal accountHoldBalances;
     uint256 override public totalSupplyOnHold;
 
     modifier isHeld(bytes32 holdId) {
         require(
-            holds[holdId].status == HoldStatusCode.Held,
+            holds[holdId].status == HoldStatusCode.Ordered,
```

Comment:
Should this be "Hold is not in Ordered status"?

Reply:
@ecp4224 we could go for @aridder 's suggestion. What do you think?

Reply:
I meant as a suggestion for the revert message

Reply:
Revert message updated